mlflow/store/model_registry/abstract_store.py

```diff
@@ -321,6 +323,37 @@ def get_model_version_by_alias(self, name, alias):
         """
         pass
 
+    @abstractmethod
+    def copy_model_version(self, src_mv, dst_name):
+        """
+        Copy a model version from one registered model to another as a new model version.
+
+        :param src_mv: A :py:class:`mlflow.entities.model_registry.ModelVersion` object representing
+                       the source model version.
+        :param dst_name: the name of the registered model to copy to.
```

Comment:
Can we describe what happens if `dst_name` doesn't exist?

Reply:
Will do!